GUI.py

```diff
@@ -355,8 +355,11 @@ def __init__(self):
         self.playButton = Button(self.master, text='Play', command=lambda: self.playback_click())
         self.playButton.grid(row=0, column=3, padx=2, pady=2)
 
+        self.pauseButton = Button(self.master, text='Pause', command=lambda: self.pause())
```

Comment:
Will need to add this function later, but good idea to add this button for usability.